R/00_generics.R

```diff
@@ -22,8 +22,8 @@
 #' If \code{NULL}, indicates all levels for a factor variable, defaults to \code{NULL}
 #' @param dof integer, the degrees of freedom used for the T statistic in an
 #' OLS model. Defaults to NULL in which case \code{mod$df.residual} is used.
-#' @param data data.frame that margins should run over, defaults to
-#' \code{mod$data}
+#' @param data data.frame that margins should run over. For glm method, defaults to
```

Comment:
Can you create an issue that we should investigate better ways of writing the documentation? It would be nice if `?marg` and `?marg.glm` and `?marg.ivreg` all led to different places?